### PR TITLE
dependency update and code formatting

### PR DESCRIPTION
**Pull request by mundra-ankur**

### CODE REVIEW

File: src/sheet.py [hunk 1]
```diff
@@ -47,29 +48,29 @@ def _column_to_letter_identifier(self, column_id):
...
+            spreadsheetId=self.sheet_id,
+            range=self.get_custom_range(start, end)
+        ).execute()['values']
 
     def read_spreadsheet(self, indexField=None):
         return self.from_sheet_data(self.client.service.spreadsheets().values().get(
-                spreadsheetId=self.sheet_id, 
-                range=self.get_sheet_range()
-            ).execute()['values'], indexField)
-
```
Comment (JonahSussman) [inline src/sheet.py]: Indenting the return statement like this emphasizes that it is a multi-line return statement. Removing it causes it to be less clear in my opinion

File: src/sheet.py [hunk 2]
```diff
@@ -84,37 +85,37 @@ def save_data_to_sheet(self, rows):
...
             ).execute()
         )
         return responses
 
     def clear_previous_data(self):
         return self.client.service.spreadsheets().values().clear(
-                spreadsheetId=self.sheet_id, 
-                range=self.get_sheet_range()
-            ).execute()
-
```
Comment (JonahSussman) [inline src/sheet.py]: Similar for removing this indentation

File: src/sheet.py [hunk 3]
```diff
@@ -177,25 +179,25 @@ def format_sheet():
     # the key is not included in the final dict.
     def sheetRange(id, r_start, r_end, c_start, c_end):
         output = {}
-        if id      is not None: output['sheetId'] = id
+        if id is not None: output['sheetId'] = id
         if r_start is not None: output['startRowIndex'] = r_start
-        if r_end   is not None: output['endRowIndex'] = r_end
+        if r_end is not None: output['endRowIndex'] = r_end
         if c_start is not None: output['startColumnIndex'] = c_start
-        if c_end   is not None: output['endColumnIndex'] = c_end
```
Comment (JonahSussman) [inline src/sheet.py]: Justifying this makes it clearer as to what the code is doing. Less mental energy to see the structure in my opinion

File: src/sheet.py [hunk 4]
```diff
@@ -84,37 +85,37 @@ def save_data_to_sheet(self, rows):
...
-        body = { 'values': [ [ 'Updated On', '{}'.format(now) ] ] }
+
+        body = {'values': [['Updated On', '{}'.format(now)]]}
 
         return self.client.service.spreadsheets().values().update(
-                spreadsheetId=self.sheet_id, 
-                range="{}!{}".format(self.sheet_name, 'A3'),
-                valueInputOption='USER_ENTERED', 
-                body=body
-            ).execute()
```
Comment (JonahSussman) [inline src/sheet.py]: Similar for removing this indentation

File: src/sheet.py [hunk 5]
```diff
@@ -177,25 +179,25 @@ def format_sheet():
...
+        return {'backgroundColor': {'red': r, 'green': g, 'blue': b}}
 
     # Returns a dict with key 'textFormat' set to the values in the params
     def textFormat(size, bold):
-        return { 'textFormat': { 'fontSize': size, 'bold': bold } }
+        return {'textFormat': {'fontSize': size, 'bold': bold}}
 
     # Returns a dict with key 'numberFormat' set to the values in the params
     def numberFormat(type, pattern):
-        return { 'numberFormat': { 'type': type, 'pattern': pattern } }
```
Comment (JonahSussman) [inline src/sheet.py]: Curly braces without spaces makes me sad :(

**Review by JonahSussman — CHANGES_REQUESTED** [review]

**Review by chaitanyab2311 — APPROVED** [review]

**Pull request merged**